src/analyzers/staged-analysis-engine.ts

```
@@ -0,0 +1,2086 @@
+import { Project, Node, TypeChecker, CallExpression, NewExpression, SourceFile, Symbol, ImportDeclaration, ModuleResolutionKind, ClassDeclaration, MethodDeclaration, PropertyAccessExpression } from 'ts-morph';
```

Biome (1.9.4)

[error] Do not shadow the global "Symbol" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)